.travis.yml

```diff
@@ -38,7 +38,7 @@ jobs:
               - beaver dlang make pkg
           deploy:
               provider: script
-              script: beaver bintray upload build/last/pkg/*.deb
+              script: beaver bintray upload build/last/pkg/*.deb -d dlang-community/apt/dfmt
```

Comment:
the option should go before the arguments: `beaver bintray upload -d dlang-community/apt/dfmt build/last/pkg/*.deb`